---
layout: paper
title: p53motifDB - integration of genomic information and tumor suppressor p53 binding motifs
image: /images/papers/2024-baniulyte-database.png
authors: Baniulyte G, Hicks SM, and Sammons MA
ref: Baniulyte G, Hicks SM, and Sammons MA. bioRxiv 2024
journal: "bioRxiv"
pdf: /pdfs/papers/2024-baniulyte-database.pdf
doi: 10.1101/2024.09.24.614594  
tag: p53
---

# Abstract

The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.

### Access the Database
[p53motifDB](https://p53motifDB.its.albany.edu)