<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8"/>
	<title>Sammons Lab / Links</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
	<!-- RSS feed -->
	<link rel="alternate" type="application/rss+xml" title="Sammons Lab Research" href="http://sammonslab.org/feed.xml">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/albany-logo.png"/>

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>

	
	    
	<!-- Google Analytics -->	
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-63799226-1', 'sammonslab.org');
		ga('send', 'pageview');

	</script>
	
	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>
	
	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>

</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-expand-sm navbar-light bg-navbar">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block align-top ml-0 mr-1" src="/images/nuc.svg">
					sammons lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link">
						
						<a class="mx-1" href="/research/">research</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/team/">team</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/papers/">papers</a></li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/news/">news</a></li>
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/links/">links</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container mt-4">

	<div class="row">
	<div class="col-md-12">
		<div class="centered-pills">
			<ul class="nav nav-pills note">
				<li role="presentation" class="active">
					<a class="off" href="/links/">
						<i class="fa fa-refresh fa-spin"></i> Timeline
					</a>
				</li>
				<li role="presentation">
					<a class="off" href="/links/archive/">
						<i class="fa fa-list-ul fa-fw"></i> Index
					</a>
				</li>
			</ul>
		</div>
		<div class="bigspacer"></div>
	</div>
</div>

<div class="post-list">



	

	<div class="blog-post">
		<div class="row">
			<div class="col-md-1"></div>	
			<div class="col-md-10">
				<div class="media">
					<div class="media-body">
						<div class="title media-heading">
							<a href="/links/RNA-Workshop/" class="off">Sequencing Technology Workshop Information - 2019</a>
						</div>
						<p>
						<div class="smallhead">
							22 Mar 2019
							
							by <a class="off" href="/team/morgan-sammons/">sammons</a>
									
						</div>
					</div>
				</div>
			</div>
			<div class="col-md-1"></div>					
		</div>

		<div class="bigspacer"></div>

		<div class="row">
			<div class="col-md-1"></div>
			<div class="col-md-10">	
				
				<div class="post">
					<h3 id="information-for-attendees-of-the-sequencing-technology-workshop-2019">Information for Attendees of the Sequencing Technology Workshop, 2019</h3>
<h4 id="part-of-the-6th-annual-rna-symposium-of-the-rna-institute">Part of the 6th Annual RNA Symposium of the RNA Institute</h4>

<p><strong>Instructor:</strong> <a href="/team/Morgan-Sammons/">Morgan Sammons</a>, Assistant Professor of Biology, State University of New York at Albany</p>

<p><strong>Instructor:</strong> Ryan Meng, Bioinformatics Support Specialist, State University of New York at Albany</p>

<blockquote>
  <p>There are a number of resources online for learning about and using high-throughput sequencing data in your own work. The <a href="https://hbctraining.github.io/main/">Harvard Chan Bioinformatics Core</a> provides extremely detailed, well-organized introductions to a number of sequencing approaches, software, and workflows. I highly recommend this for further information.</p>
</blockquote>

<dl>
  <dt><a href="/pdfs/links/rnasymp/Sammons.RNAsymp.SeqTech.Workshop.2019.pdf">Main Presentation</a></dt>
  <dd>Presentation given by Morgan as part of the workshop.</dd>
  <dt><a href="/pdfs/links/rnasymp/Schurch.2016.RNA.pdf">Bulk RNA-seq Replicate Guidelines</a></dt>
  <dd>Excellent manuscript published in <em>RNA</em> discussing why biological replicates matter and how to select the number of replicates in a bulk RNA-seq experiment.</dd>
</dl>

<h4 id="tools-used-during-the-symposium">Tools used during the symposium</h4>
<p><a href="https://github.com/alexdobin/STAR">STAR</a>: a splice-aware alignment tool.</p>

<blockquote>
  <p>Dobin, A., Davis, C. A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., … Gingeras, T. R. (2012). STAR: ultrafast universal RNA-seq aligner. Bioinformatics (Oxford, England), 29(1), 15-21. doi: <a href="10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a></p>
</blockquote>

<p><a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a>: An <em>R</em>-based package for differential gene expression analysis using raw read counts derived from STAR. <a href="http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html">Short Tutorial</a></p>

<blockquote>
  <p>Love, M.I., Huber, W., Anders, S. (2014) Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biology, 15:550 doi: <a href="10.1186/s13059-014-0550-8">10.1186/s13059-014-0550-8</a></p>
</blockquote>

<p><a href="http://www.htslib.org/">Samtools</a>: software for manipulating SAM and BAM alignment files.</p>

<blockquote>
  <p>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, and 1000 Genome Project Data Processing Subgroup, The Sequence alignment/map (SAM) format and SAMtools, Bioinformatics (2009) 25(16) 2078-9 doi: <a href="10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></p>
</blockquote>

<h4 id="commands-used-for-genome-indexing-and-alignment">Commands used for Genome Indexing and Alignment</h4>

<p><a href="/pdfs/links/rnasymp/STARIndex.bash">Genome Indexing with STAR</a></p>

<p>We used the <em>Saccharomyces cerevisiae</em> sacCer3 genome from UCSC found at the <a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html">Illumina iGenomes</a> website.</p>

<p>iGenomes is a nice source for the genomes of many model organisms used in research.</p>

<p>You can use STAR (or other genome aligners) to build indexes to your own model organism using a FASTA file of the genome.</p>

<p><a href="/pdfs/links/rnasymp/STARAlign.bash">Genome Alignment with STAR</a></p>

<p>We used STAR in a cluster setting to align our raw FASTQ files to the sacCer3 genome we indexed in the prior steps.</p>

<p>The script shown here is designed to be used in a cluster setting using the <a href="https://slurm.schedmd.com/tutorials.html">slurm</a> schedule manager. If you are performing these tasks locally, you would not need to use the slurm nomenclature. If you are performing these tasks on your home institution cluster, they may use another schedule manager.</p>

<p>Another point to remember is that we asked for certain computational resources in order to do the alignment. We based these numbers off of the available computational resources and our expectations of how long the alignment should take based on it being yeast (versus human or mouse with larger genomes).</p>

<h4 id="non-alignment-based-strategies-for-differential-gene-expression">Non-alignment Based Strategies for Differential Gene Expression</h4>

<p>We used <a href="https://salmon.readthedocs.io/en/latest/">salmon</a> to perform transcript quantification of our RNA-seq data without prior alignment to a reference genome.</p>

<p>The major advantage over STAR (or other aligners) is <strong>speed</strong>. salmon and a similar tool called <a href="https://pachterlab.github.io/kallisto/">kallisto</a> do not actually perform any alignments, which drastically speeds up the process of quantifying your RNA-seq data.</p>

<p>The other huge advantage is the processing power required to perform the transcript quantification using salmon or kallisto. These two programs can be run on your laptop or desktop and do not require anything more than that!</p>

				</div>	
			</div>			
			<div class="col-md-1"></div>
		</div>
	
		<div class="bigspacer"></div>
		<div class="spacer"></div>	

	</div>



	

	<div class="blog-post">
		<div class="row">
			<div class="col-md-1"></div>	
			<div class="col-md-10">
				<div class="media">
					<div class="media-body">
						<div class="title media-heading">
							<a href="/links/BIO681_Spring2019/" class="off">BIO681 - Seminar in MCDN - Reading List</a>
						</div>
						<p>
						<div class="smallhead">
							25 Jan 2019
							
							by <a class="off" href="/team/morgan-sammons/">sammons</a>
									
						</div>
					</div>
				</div>
			</div>
			<div class="col-md-1"></div>					
		</div>

		<div class="bigspacer"></div>

		<div class="row">
			<div class="col-md-1"></div>
			<div class="col-md-10">	
				
				<div class="post">
					<h3 id="information-for-abio681---spring-2019">Information for ABIO681 - Spring 2019</h3>

<p><a href="/pdfs/bio681/ABIO681_MCDNSeminar_GenomescaleBio_Spring2019_Sammons.pdf">Syllabus</a></p>

<hr />

<dl>
  <dt><a href="/pdfs/bio681/Casadevall_and_Fang_2009.pdf">Reading due 02/05</a></dt>
  <dd>Casadevall and Fang 2009</dd>
</dl>

<hr />

<table>
  <thead>
    <tr>
      <th><strong>Presentation Date</strong></th>
      <th><strong>Paper</strong></th>
      <th><strong>Presenters</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>12-Feb</td>
      <td><a href="/pdfs/bio681/Fields_and_Song_1989.pdf">Fields and Song 1989</a></td>
      <td>McCauley/Altreith</td>
    </tr>
    <tr>
      <td>19-Feb</td>
      <td><a href="/pdfs/bio681/Wang_1993.pdf">Wang 1993</a> and <a href="/pdfs/bio681/Wilson_1991.pdf">Wilson 1991</a></td>
      <td>Lin/Catizone</td>
    </tr>
    <tr>
      <td>26-Feb</td>
      <td><a href="/pdfs/bio681/DeRisi_1996.pdf">DeRisi et al 1996</a></td>
      <td>Koslow/McCarthy</td>
    </tr>
    <tr>
      <td>5-Mar</td>
      <td><a href="/pdfs/bio681/Burns_1994.pdf">Burns et al 1994</a> and <a href="/pdfs/bio681/Giaever_2002.pdf">Giaever et al 2002</a></td>
      <td>O’Keefe/Martin</td>
    </tr>
    <tr>
      <td>12-Mar</td>
      <td><a href="/pdfs/bio681/Krogan_2006.pdf">Krogan et al 2006</a></td>
      <td>Soyer/Sammons</td>
    </tr>
    <tr>
      <td>26-Mar</td>
      <td><a href="/pdfs/bio681/Bentley_2008.pdf">Bentley et al 2008</a></td>
      <td>Durham/Naik</td>
    </tr>
    <tr>
      <td>9-Apr</td>
      <td><a href="/pdfs/bio681/Ren_2000.pdf">Ren et al 2000</a> and <a href="/pdfs/bio681/Johnson_2007.pdf">Johnson et al 2007</a></td>
      <td>Waldern/Moskwa</td>
    </tr>
  </tbody>
</table>

<hr />
<p>| <strong>Presentation Date</strong> | <strong>Paper</strong> | <strong>Authors</strong> | <strong>Reviewers</strong> |
|———————–|———–|————-|—————|
|16-Apr	|	<a href="/pdfs/bio681/Barski_2007.pdf">Barski et al 2007</a>|O’Keefe/Naik|	Moskwa/McCarthy|
|23-Apr|	<a href="/pdfs/bio681/Korthout_2018.pdf">Korthout et al 2018</a>	 |Altreith/Martin|	Durham/Waldern|</p>

<hr />

<dl>
  <dt><a href="/pdfs/bio681/Casadevall_and_Fang_2014.pdf">Reading due 04/30 #1</a></dt>
  <dd>Casadevall and Fang 2014</dd>
  <dt><a href="/pdfs/bio681/Berg_2016.pdf">Reading due 04/30 #2</a></dt>
  <dd>Berg 2016</dd>
</dl>

<hr />

<table>
  <thead>
    <tr>
      <th><strong>Presentation Date</strong></th>
      <th><strong>Paper</strong></th>
      <th><strong>Authors</strong></th>
      <th><strong>Reviewers</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>7-May</td>
      <td><a href="/pdfs/bio681/Lee_2018.pdf">Lee et al 2019</a></td>
      <td>McCauley/Catizone</td>
      <td>Koslow/Lin</td>
    </tr>
  </tbody>
</table>


				</div>	
			</div>			
			<div class="col-md-1"></div>
		</div>
	
		<div class="bigspacer"></div>
		<div class="spacer"></div>	

	</div>



	

	<div class="blog-post">
		<div class="row">
			<div class="col-md-1"></div>	
			<div class="col-md-10">
				<div class="media">
					<div class="media-body">
						<div class="title media-heading">
							<a href="/links/WebResources/" class="off">Web Resources</a>
						</div>
						<p>
						<div class="smallhead">
							23 Dec 2017
							
							by <a class="off" href="/team/morgan-sammons/">sammons</a>
									
						</div>
					</div>
				</div>
			</div>
			<div class="col-md-1"></div>					
		</div>

		<div class="bigspacer"></div>

		<div class="row">
			<div class="col-md-1"></div>
			<div class="col-md-10">	
				
				<div class="post">
					<p>Links I really like</p>

<dl>
  <dt><a href="http://www.genome.ucsc.edu">UCSC Genome Browser</a></dt>
  <dd>Ubiquitous and fantastically data rich, UCSC Genome Browser is my go-to.</dd>
  <dt><a href="http://epigenomegateway.wustl.edu/">WashU Epigenome Browswer</a></dt>
  <dd>Excellent 3D visualization tools as well as host to a number of consortium data hubs</dd>
  <dt><a href="https://portals.broadinstitute.org/ccle">Cancer Cell Line Encyclopedia</a></dt>
  <dd>Search for your favorite genes and compare expression across multiple cancer cell lines. Very well done.</dd>
  <dt><a href="http://homer.ucsd.edu/homer/">HOMER</a></dt>
  <dd>One of the best annotated software suites, this is still my favorite semi-integrated next-gen sequencing analysis software. Bonus for being great at its original function: DNA motif analysis</dd>
  <dt><a href="http://bedtools.readthedocs.io/en/latest/">Bedtools</a></dt>
  <dd>Indispensable tool for analysis of genomic intervals/peak locations.</dd>
  <dt><a href="https://brew.sh/">Homebrew</a></dt>
  <dd>Easiest way to install and update command line software tools on Mac.</dd>
  <dt><a href="http://formulae.brew.sh/">Homebrew Formulas</a></dt>
  <dd>Nice list of software that can be installed/managed by homebrew.</dd>
  <dt><a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus</a></dt>
  <dd>NIH-funded repository for genomics data of all types</dd>
  <dt><a href="https://www.ncbi.nlm.nih.gov/geo/">Sequence Read Archive</a></dt>
  <dd>NIH-funded repository for next-generation sequencing reads.</dd>
  <dt><a href="https://deeptools.readthedocs.io/en/latest/index.html">deeptools</a></dt>
  <dd>Very nice suite of tools for some ChIP-seq/RNA-seq applications; many uses!</dd>
</dl>

<p><a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">DEseq2</a></p>

<dl>
  <dt><a href="https://github.com/alexdobin/STAR">STAR Aligner</a></dt>
  <dd>Super fast and accurate aligner: requires high RAM for mammalian genomes, but speed can’t be beat.</dd>
  <dt><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie2</a></dt>
  <dd>My favorite short read aligner (mainly because I know how to use it).</dd>
  <dt><a href="http://hocomoco11.autosome.ru/">HOCOMOCO</a></dt>
  <dd>Nice database of DNA-binding protein motifs</dd>
  <dt><a href="http://firebrowse.org/">Firebrowse</a></dt>
  <dd>Integration of (primarily) the Cancer Genome Atlas (TCGA) data</dd>
  <dt><a href="http://www.cbioportal.org/">cBioPortal</a></dt>
  <dd>Integration of TCGA and many other datasets</dd>
  <dt><a href="http://emboss.sourceforge.net/">EMBOSS</a></dt>
  <dd>Software suite that performs numerous DNA/RNA sequence manipulations</dd>
  <dt><a href="http://jaspar.genereg.net/">JASPAR</a></dt>
  <dd>Transcription factor binding motif database</dd>
  <dt><a href="https://software-carpentry.org/lessons/">Software Carpentry</a></dt>
  <dd>Learn how to code (I need to do this)</dd>
</dl>

				</div>	
			</div>			
			<div class="col-md-1"></div>
		</div>
	
		<div class="bigspacer"></div>
		<div class="spacer"></div>	

	</div>



	

	<div class="blog-post">
		<div class="row">
			<div class="col-md-1"></div>	
			<div class="col-md-10">
				<div class="media">
					<div class="media-body">
						<div class="title media-heading">
							<a href="/links/Bootcamp/" class="off">Albany NGS Bootcamp</a>
						</div>
						<p>
						<div class="smallhead">
							21 Apr 2017
							
							by <a class="off" href="/team/morgan-sammons/">sammons</a>
									
						</div>
					</div>
				</div>
			</div>
			<div class="col-md-1"></div>					
		</div>

		<div class="bigspacer"></div>

		<div class="row">
			<div class="col-md-1"></div>
			<div class="col-md-10">	
				
				<div class="post">
					<dl>
  <dt><a href="/pdfs/links/Intro_to_NextGen_Sammons_04212017.pdf">Intro to Next-Gen Sequencing</a></dt>
  <dd>A completely non-comprehensive introduction into next-gen sequencing. Primarily designed as part of a larger bootcamp held @ UAlbany</dd>
  <dt><a href="/pdfs/links/Sammons_NGS_Guidelines_April2017_WebVersion.pdf">Getting Started with Next-Gen Sequencing</a></dt>
  <dd>Tips and tricks for getting started in sequencing at UAlbany</dd>
  <dt><a href="/pdfs/links/Bootcamp_ATAC_Seq_Sammons_April2017_WebVersion.pdf">Applications: ATAC-seq</a></dt>
  <dd>Data I presented that turned into <a href="http://thesammonslab.org/papers/sammons-science/">this</a> paper with the Wherry lab @ the University of Pennsylvania</dd>
</dl>

				</div>	
			</div>			
			<div class="col-md-1"></div>
		</div>
	
		<div class="bigspacer"></div>
		<div class="spacer"></div>	

	</div>



</div>

<div class="infinite-spinner"></div>

<div class="row">
	<div class="col-md-12">
		<div id="scroll" class="centered-pills">
			<ul class="nav nav-pills note">
				<li role="presentation">
					<a class="off" href="/news/">
						<i class="fa fa-chevron-up fa-fw"></i> Back to top
					</a>
				</li>				
			</ul>
		</div>
	</div>
</div>

<!-- Infinite scroll -->	
<script async src="/js/infinite-scroll.js"></script>

	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>
